src/components/ClockModule.js

```diff
@@ -147,137 +150,145 @@ function ClockModule() {
       />
 
       <HeaderPopover
-        animateHeight={false}
         heading="Court clock"
-        height={(isSynced ? 27.5 : 26) * GU}
-        width={45 * GU}
```

Comment:
Wasn't sure if we wanted to keep these original dimensions, but the clock popover still looks good with the `30 x 51` GU dimensions.

Reply:
I re-introduced the old Popover again; should we update to the new `30 x 51` GU dimensions anyway?

Reply:
No, let's keep it as it was!